packages/components/src/assets/CircleWavyCheck.tsx

```diff
@@ -0,0 +1,31 @@
+import type { FC } from 'react'
+import React from 'react'
+
+// Icon from Phosphor Icons
+const CircleWavyCheck: FC = () => (
```

Comment:
How about take the opportunity and uses the new approach (using IconProps) for icons assets?

https://github.com/vtex/faststore/blob/main/packages/components/src/assets/House.tsx

Reply:
I think this can be done in another task for all icons.

Reply:
If it's not that complicated, we can use the new approach here already ;)

Reply:
Nice! done [here](https://github.com/vtex/faststore/pull/1672/commits/c2ec26c8b4740bdbee0a9312283327aa0b1425b6).